app/javascript/react/components/Greetings.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGreetings } from '../redux/greetings/greetings';
+
+const Greetings = () => {
+/* Load Redux State */
+  const greetings = useSelector((state) => state.greeting);
+  // *********************************
+
+  /* Setup Redux dispatch */
+  const dispatch = useDispatch();
+
+  /* Clean - Redux movies store */
+  useEffect(() => {
+    dispatch(getGreetings());
+  }, [dispatch]);
+
+  return (
+    <>
+      <h2>Random Greetings</h2>
+      <p>{ greetings }</p>
+    </>
+  );
+};
+
+export default Greetings;
```

Comment:
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/65068771/228642084-34fcb6ae-01ce-491e-a424-8766642605b0.png">

- [ ] Make sure the A greeting message is rendered here coming from the database. I have already created the gretting in the database however now gretting is display on the react component.

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/65068771/228641830-30e6e46b-a293-4961-8913-3ea22329634f.png">